src/components/configure/ExportImport.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { Upload, message, Form, Select, Input, Button } from 'antd';
+import { Icon } from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import { setProjectConfig, getProjectConfig, notify } from '../../utils';
+import { useParams } from "react-router-dom";
+import { get, set, increment, decrement } from 'automate-redux';
+import client from "../../client"
+
+
+const YAML = require('yamljs');
+const yaml = require('js-yaml');
+
+const ExportImport = ({ object }) => {
+    const { projectID, selectedDB } = useParams()
+    const [Type, setType] = useState('');
+    const dispatch = useDispatch()
+
+
+    const projects = useSelector(state => state.projects)
+    const projectName = getProjectConfig(projects, projectID, "name")
+
+    const downLoadJson = e => {
+        e.preventDefault();
+
+        const data = new Blob([JSON.stringify(object)], { type: 'content-type/json' });
+        const jsonURL = window.URL.createObjectURL(data);
+        const jsonLink = document.createElement('a');
+        jsonLink.href = jsonURL;
+        jsonLink.setAttribute('download', 'config.json');
+        jsonLink.click();
+
+    }
+    const downLoadYaml = e => {
```

Comment:
Maybe we can create a single file for downloading json and yaml file since the majority of the code is similar?